tests.py

```diff
@@ -176,3 +205,66 @@ def test_amend_requirements_content_replacement_amonst_others_2(self):
             requirements, 'autocompeter', new_lines
         )
         self.assertEqual(result, previous + new_lines)
+
+    @httpretty.activate
+    @cleanup_tmpdir('peepin*')
+    def test_run(self):
+        httpretty.register_uri(
+            httpretty.GET,
+            "https://pypi.python.org/pypi/peepin",
+            body="""
+            <div id="content">
```

Comment:
Doesn't matter. It's a regex that looks for [certain hrefs](https://github.com/peterbe/peepin/blob/master/peepin.py#L117)